packages/default-evaluate-options/package.json

```diff
@@ -23,10 +24,8 @@
   },
   "dependencies": {
     "@agoric/babel-parser": "^7.6.4",
-    "@agoric/eventual-send": "^0.4.4",
-    "@agoric/transform-eventual-send": "^1.0.3",
-    "@babel/generator": "^7.6.4",
-    "esm": "^3.2.5"
+    "@agoric/transform-eventual-send": "^1.0.4",
```

Comment:
this asks for `transform-eventual-send` 1.0.4, but that package was upgraded from `1.0.3` to `1.1.0` . Should this be changed to require `1.1.0`? Or can `default-eventual-options` use the previous version without problems?

Reply:
It should be changed.  Thanks.